lib/tasks/update_database_records_in_big_query.rake

```diff
@@ -2,7 +2,7 @@ namespace :database_records do
   desc 'Updates Big Query Datasets with latest database records'
   namespace :in_big_query do
     task update: :environment do
-      ExportVacancyRecordsToBigQueryJob.perform_later
+      # ExportVacancyRecordsToBigQueryJob.perform_later
```

Comment:
Can you please remove these rather than commenting them out?  They'll still be in the history if we need them.